ssl/quic/quic_record_rx.c

```diff
@@ -757,12 +757,25 @@ static int qrx_decrypt_pkt_body(OSSL_QRX *qrx, unsigned char *dst,
     if (EVP_CipherUpdate(cctx, dst, &l, src, src_len - el->tag_len) != 1)
         return 0;
 
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
+    /*
+     * Throw away what we just decrypted and just use the ciphertext instead
+     * (which should be unencrypted)
+     */
+    memcpy(dst, src, l);
+
+    /* Pretend to authenticate the tag but ignore it */
+    if (EVP_CipherFinal_ex(cctx, NULL, &l2) != 1) {
```

Comment:
nit: Don't we normally cast this as:
(void)EVP_CipherFinal_ex(cctx, NULL, &l2)

Reply:
Strangely, because this function is decorated with `__owur`, even casting to void isn't sufficient to suppress the warning about an unused result. You actually have to really do something with it.

Reply:
In other circumstances I have worked around this issue using this construction:

```c
static void ignore_res(int x) {}

ignore_res(foo());
```

I feel like this is at least a bit nicer.